api/src/Migrations/Version268.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version268 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```

Comment:
Sorry kinda trivial, but could you add a description here?

Reply:
Good shout, totally missed this!